core/admin/mailu/sso/views/base.py

```diff
@@ -9,9 +9,33 @@
 import secrets
 import ipaddress
 
+from oic import rndstr
+
 @sso.route('/login', methods=['GET', 'POST'])
```

Comment:
IMHO the openid endpoint should be different from the "standard" login flow: use a different route!

Reply:
done, changed to `/login/oic` for the callback url